app/Http/Controllers/HomeController.php

```diff
@@ -17,4 +17,16 @@ public function index(TimeLogRepository $timeLogRepository)
         $weekly_summary = $timeLogRepository->weeklySummary();
         return view('welcome', compact('weekly_summary'));
     }
+
+     /**
+     * Show the profile config page.
+     *
+     * @param  App\Modules\Time\Repositories\Interfaces\TimeLogRepositoryInterface $timeLogRepository
+     * @return \Illuminate\Http\Response
+     */
+    public function indexEmployee(TimeLogRepository $timeLogRepository)
+    {
+        $weekly_summary = $timeLogRepository->weeklySummary();
```

Comment:
This part should not be here at the moment. The current feature should only cover the employee login, we don't need any other functionality.